app/support/auto_expire_reservation.rb

```diff
@@ -15,7 +15,8 @@ def order_details
 
   def purchased_active_order_details
     OrderDetail.purchased_active_reservations
-      .merge(Reservation.relay_in_progress)
+      .joins("INNER JOIN products ON order_details.product_id = products.id")
+      .joins("INNER JOIN relays ON relays.instrument_id = products.id")
```

Comment:
Would it make sense to create a scope--thinking something like `usuage_based` on products that checks that `control_mechanism != Relay::CONTROL_MECHANISMS[:manual]`?